### ISSUE TITLE
Three new positions for PhD candidates at HU Berlin!

### ISSUE THREAD

**Issue by alanakbik** [flairNLP/flair#2446]
Hi all, 

as mentioned earlier (see #2342), my research group at HU Berlin is growing quickly. 

We now have **three more positions for PhD candidates** becoming available! The positions are full-tme and fully paid research associate positions in my group, for persons with a master degree to do research and pursue a PhD in machine learning or NLP: 
- one position to begin at the end of 2021
- two positions to begin in October 2022 _(but the application deadline is already November 2021!)_
    
The first is at the intersection of NLP and ML. The second two are core ML positions in which we look to overcome fundamental problems in deep learning and so require in-depth knowledge of ML. 

For all three positions, the deadline is soon (October/November). **Consider applying** and contact me in case of interest! I'll also post more information once the job ads are up.

Cheers, 
Alan

**Comment by Dashulkin** [comment]
Dear @alanakbik,
could you please send a bit more information with apply form or something? I would like to apply and try to enter the community. Really like the atmosphere and things happen here. 
Thank you in advance!

Best regards, 
Daria Gavrilik

**Comment by alanakbik** [comment]
Hello @Dashulkin I will add the forms once they are up! More information on the two positions for 2022 will likely be posted here on October 25th. Information on the position that starts end of year will be posted soon(ish), likely in the next weeks or so.

**Comment by alanakbik** [comment]
Hi all, a first update on this: The **two positions for October 2022** are now posted! Application deadline is **November 25th 2021**!

Here the important links:

- The first is in the research project "**Efficient Model Learning from Data with Partially Incorrect Labels**". Essentially the question is: how can we learn good NLP models even if the training data is not perfect (for instance if some labels are wrong or missing)? This topic is at the intersection of ML and NLP, and may include topics like modelling uncertainty, semi-supervised learning, etc. Link here: https://scioi.softgarden.io/job/13424722?l=en 

- The second is the research project: "**Modeling Neurogenesis for Continuous Learning**". Here the question is whether the structure/topology of neural networks themselves can be learnt (instead of being manually specified by a data scientist). A particular focus is on continuous learning and combating catastrophic forgetting. Its more of an ML topic, but with applications in NLP continuous learning. Link here: https://scioi.softgarden.io/job/13423947?l=en

(In the second project we collaborate with the robotics group of Prof. Hafner that looks at this question more from a robotics side. If you're interested more in this, there is a position in her group for the same project (so we'd collaborate a lot): https://scioi.softgarden.io/job/13424162?l=en)

In total, there are 11 doctoral positions now becoming available in the "Science of Intelligence" cluster of excellence here in Berlin (two directly in my group, the others supervised by other groups in Berlin). A full overview is here: https://www.scienceofintelligence.de/call-for-applications/open-positions/ 

Consider applying! And do contact me if you have any questions!

**Comment by alanakbik** [comment]
Hi all, **another PhD position** is now posted, for the **beginning of 2022**. Deadline for applications is November 24th!

The position posting is [here](https://www.personalabteilung.hu-berlin.de/de/stellenausschreibungen/wissenschaftliche-r-mitarbeiter-in-m-w-d-mit-vorauss-vollzeit-e-13-tv-l-hu-drittmittelfinanzierung-befristet-bis-31-12-2023). But since the posting is in German, in the following the key info:

The position is in a joint research project with search engine specialist Neofonie GmbH and will involve research in the broad area of learning good representations of natural language from text data. This may involve research in neural language modeling and information retrieval. Your tasks will be to pursue such research at the intersection of NLP/ML with the goal of attaining a PhD. And of course help out in the group for instance by helping develop Flair and giving guidance to bachelor/master students. 

Requirements:
- completed masters degree in computer science or computational linguistics
- relevant prior knowledge in the areas of Natural Language Processing (NLP) and Deep Learning
- very good Python knowledge
- experience with Deep Learning frameworks like PyTorch
- desirable: first publications in the field of NLP, collaboration in open source projects

Consider applying! Application is directly to me via mail. Please include your GitHub ID and the posting number (DR/169/21) in the application.

**Comment by otahmasebi** [comment]
Hi @alanakbik,
Although you said that English is not required! in the link you share, proof of English skills is a must. what does it mean?! Should the TOEFL/IELTS score be included?!

**Comment by alanakbik** [comment]
Oh oops, I meant to type "German is not required" when I answered @IMTorg 's question whether one needs German. I'll edit my above response (sorry for the confusion!).  

English is indeed needed for the two positions in SCIoI. The following links give additional information on how to apply for these positions: 
- Very helpful FAQs (we've received many questions): https://www.scienceofintelligence.de/education/admissions/admissions-faqs/ 
- Overview of admission process: https://www.scienceofintelligence.de/education/admissions/ 

Any other questions regarding the application process should be directed to applications@scioi.de

**Comment by varma1131987** [comment]
I am Interested , please help out the process how to move forward

**Comment by abdullahmuaad9** [comment]
could you please send a bit more information with apply form or something? I would like to apply and try to enter the community. 
now we are around April 2022 is there any updation
Thank you in advance

**Comment by alanakbik** [comment]
A new position is available! See #2874!
